content/docs/user-guide/experiment-management/index.md

```diff
@@ -12,16 +12,22 @@ optimization, etc. DVC helps you codify and manage all of your
    version without having to track them manually. DVC does that automatically,
    letting you list and compare them. The best ones can be made persistent, and
    the rest archived.
+
 2. Place in-code [checkpoints](#checkpoints-in-source-code) that mark a series
    of variations, forming a deep experiment. DVC helps you capture them at
    runtime, and manage them in batches.
+
 3. Make experiments or checkpoints [persistent](#persistent-experiments) by
    committing them to your <abbr>repository</abbr>. Or create these versions
    from scratch like typical project changes.
 
-   At this point you may also want to consider the different
-   [ways to organize](#organization-patterns) experiments in your project (as
-   Git branches, as folders, etc.).
+   > At this point you may also want to consider the different
+   > [ways to organize](#organization-patterns) experiments in your project as
+   > well.
+
+4. Easily [share experiments] across your team.
```

Comment:
minor: I think it's usually good not to use `easy` and similar terms

Reply:
Changed to "seamlessly" for now. We kind of need a qualifier here I think.